test/core/simd/meta/simd_f64x2.py

```diff
@@ -0,0 +1,443 @@
+                                                                            ''))
+                else:
+
+                    lst_diff_lane_vs_clause_assert.append(tpl_assert.format('assert_return',
+                                                                            tpl_func_name_by_lane.format(lane_idx),
+                                                                            self.v128_const('f64x2', case_data[1][0]),
+                                                                            self.v128_const('f64x2', case_data[1][1]),
+                                                                            self.v128_const('i32', idx_func),
+                                                                            '  '+self.v128_const('f64', ret)+'\n'))
+
+            case_cnt += 1
+            if case_cnt == 2:
+                case_cnt = 0
+
+        lst_diff_lane_vs_clause_assert.append('')
+
+        # Add test for operations with constant operands
+        for key in lst_oprt_with_const_assert:
+
+            case_cnt = 0
+            for case_data in lst_oprt_with_const_assert[key]:
+
+                # Add comment for the param combination
+                if case_cnt == 0:
+                    template.append(';; {} const vs const'.format(op_name))
+                if case_cnt == 4:
+                    template.append(';; {} param vs const'.format(op_name))
+
+                # Cross parameters and constants
+                if case_cnt < 8:
+                    template.append(str(AssertReturn(case_data[0], [], self.v128_const('f64x2', case_data[1]))))
+                else:
+                    template.append(str(AssertReturn(case_data[0], [self.v128_const('f64x2', case_data[1])], self.v128_const('f64x2', case_data[2]))))
+                case_cnt += 1
+
+        # Generate and append f64x2.abs assert
+        op_name = self.full_op_name('abs')
+        template.append('')
+        func_name = "{}_with_const_{}".format(op_name, 35)
+        template.append(str(AssertReturn(func_name, [], self.v128_const('f64x2', ['0', '1']))))
+        func_name = "{}_with_const_{}".format(op_name, 36)
+        template.append(str(AssertReturn(func_name, [], self.v128_const('f64x2', ['2', '3']))))
+
+        template.extend(lst_diff_lane_vs_clause_assert)
+
+        return template
+
+    @property
+    def combine_ternary_arith_test_data(self):
+        return {
+            'min-max': [
+                ['1.125'] * 2, ['0.25'] * 2, ['0.125'] * 2, ['0.125'] * 2
+            ],
+            'max-min': [
+                ['1.125'] * 2, ['0.25'] * 2, ['0.125'] * 2, ['0.25'] * 2
+            ]
+        }
+
+    @property
+    def combine_binary_arith_test_data(self):
+        return {
+            'min-abs': [
+                ['-1.125'] * 2, ['0.125'] * 2, ['0.125'] * 2
+            ],
+            'max-abs': [
+                ['-1.125'] * 2, ['0.125'] * 2, ['1.125'] * 2
+            ]
+        }
```

Comment:
I'm not sure how these structures are used either. They seem different from the previous structures.

Reply:
These structures inherit from [simd_arithmetic.py](https://github.com/WebAssembly/simd/blob/7abe171dab76043a335282c4a447a873e471b64f/test/core/simd/meta/simd_arithmetic.py#L241), specific for combination tests. Here should have annotate to explain this.